src/world/conquest_system.h

```diff
@@ -38,12 +38,10 @@ class ConquestSystem : public IMessageHandler
 
     /**
      * IMessageHandler implementation. Used to handle messages from message_server.
-     * NOTE: The copy of payload here is intentional, since these systems will eventually
-     *     : be moved to their own threads.
      */
-    bool handleMessage(std::vector<uint8> payload,
-                       in_addr            from_addr,
-                       uint16             from_port) override;
+    bool handleMessage(const std::vector<uint8>& payload,
+                       in_addr                   from_addr,
```

Comment:
Most of the rest of the code base (aside from `const char*` - bane of my life) is `type const&`

Reply:
I knew it was something like that! And I can never remember the right one... thanks